lms/static/scripts/postmessage_json_rpc/server/index.js

```diff
@@ -0,0 +1,9 @@
+/** The entry point for the postmessage_json_rpc_server bundle. */
+"use strict";
+
+import Server from "./server"
+import { requestConfig } from "./methods"
+
+const serverConfig = JSON.parse(document.getElementsByClassName("js-rpc-server-config")[0].textContent);
```

Comment:
`document.querySelector(".class-name")` is a slightly shorter way of writing `documents.getElementsByClassName("class-name")[0]`.